### PR TITLE
Remove web dependency (mostly)

### PR DESCRIPTION
**Pull request by YouKnowBlom**
Plan is to remove our dependency to web. Datetime and browserdeviceprofile should be fairly straight forward while fetchhelper (included in jellyfin-apiclient-javascript) will take a lot more work, probably better fit for another PR.

**Done:**
- Datetime
- Browserdeviceprofile

### CODE REVIEW

File: helpers.js [hunk 1]
```diff
@@ -901,7 +901,7 @@ function getMiscInfoHtml(item, datetime) {
 
         if (item.Type == "Audio") {
 
-            miscInfo.push(datetime.getDisplayRunningTime(item.RunTimeTicks));
+            miscInfo.push(getDisplayRunningTime(item.RunTimeTicks));
```
Comment (dkanada) [inline helpers.js]: Does this method exist elsewhere?

Comment (YouKnowBlom) [reply helpers.js]: Good catch, I was a bit too fast in my cleaning frenzy. I'll throw it into helpers.js.

**Comment by ferferga** [comment]
What if you go and use date-fns as well here? We plan to move all the datetime functions to it, so I think it makes sense to do it in Chromecast as well, as I suppose it's possible to do it, like in Web.

Cc @MrTimscampi

**Comment by YouKnowBlom** [comment]
I'll have to look into that. Thanks for the heads up!

**Pull request merged**